layouts/layout-vertical.php

```diff
@@ -27,19 +27,19 @@ function ucf_spotlight_display_vertical( $content='', $item, $args ) {
 		</div>
 		<?php endif; ?>
 
-		<?php if ( $args['link_url'] ): ?>
-		</a>
-		<?php endif; ?>
-
 		<?php if ( $args['copy'] ): ?>
 		<div class="spotlight-copy">
 			<?php echo $args['copy']; ?>
 		</div>
 		<?php endif; ?>
 
 		<?php if ( $args['link_url'] && $args['link_text'] ): ?>
-		<a class="spotlight-btn btn btn-primary mt-3" href="<?php echo $args['link_url']; ?>">
+		<div class="spotlight-btn btn btn-primary mt-3" href="<?php echo $args['link_url']; ?>">
```

Comment:
Did you intend to put the href attribute on the div?

Reply:
Nope...thanks.  Thought I got rid of that.